### PR TITLE
OrderedList

### PR DESCRIPTION
**Pull request by albertocuestacanada**
An implementation for an ordered list, it reuses DoubleLinkedList which makes it a bit confusinf in the the head is the lowest rank and the tail the highest. However that preserves the notion that it you take an object, the previous one has a lower rank and the next one a higher rank.

### CODE REVIEW

**Comment by obernardovieira** [comment]
Why isn't this merged yet?

**Comment by albertocuestacanada** [comment]
@obernardovieira, I think that only you can merge this.

**Comment by obernardovieira** [comment]
> @obernardovieira, I think that only you can merge this.

oh crap! bad config. Sorry, solved.

File: contracts/lists/OrderedList.sol [hunk 1]
```diff
@@ -0,0 +1,229 @@
+pragma solidity ^0.5.0;
```
Comment (obernardovieira) [inline contracts/lists/OrderedList.sol]: We should go with `pragma solidity ^0.5.10;`

**Pull request merged**